Build/Projects/MonoGame.Framework.definition

```diff
@@ -292,7 +292,7 @@
     <Compile Include="DisplayOrientation.cs" />
     <Compile Include="DrawableGameComponent.cs" />
     <Compile Include="FrameworkDispatcher.cs">
-      <Platforms>_FrameworkDispatcherProvided</Platforms>
+      <Services>_FrameworkDispatcherProvided</Services>
```

Comment:
> Yeah, I know that's what's supposed to happen, but that wil reintroduce the stutter 

I would still hook up `FrameworkDispatcher.Update()` at least so that the AL platforms work correctly.

It is ok for now if the XAudio path just doesn't do anything under it.